specification/agrifood/data-plane/Microsoft.AgFoodPlatform/preview/2022-11-01-preview/agfood.json

```diff
@@ -21973,6 +21973,14 @@
         "data": {
           "$ref": "#/definitions/SatelliteData"
         },
+        "providerClientId": {
```

Comment:
order of parameters should be lexicographically.

Reply:
That will lead to a lot of changes, not doing in this PR.